Anzee/Requests/CampaignRequest.swift

```diff
@@ -13,7 +13,7 @@ struct CampaignRequest: APIRequest {
 
     var params: [URLQueryItem]?
 
-    var apiPath: String { return "/campaigns/\(campaignID)" }
+    var apiPath: String { return "campaigns/\(campaignID)" }
 
     public typealias CampaignRequestCallback = ((_ campaign: Campaign?, _ error: APIError?) -> Void)
```

Comment:
@michael-patzer Would it be too much to have these use a `Result` type too?

Reply:
@veader Good idea. PR updated.